src/index.html

```diff
@@ -11,12 +11,111 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
```

Comment:
The `href` attribute value seems to be incomplete. It should be a full URL to the resource you're trying to preconnect to.